wp-content/themes/core/assets/pcss/_mixins.pcss

```diff
@@ -0,0 +1,14 @@
+/* -------------------------------------------------------------------------
+ *
+ * Mixins
+ *
+ * This file is just a clearing-house.
+ * Make partials (start with an underscore) elsewhere for actual code.
+ *
+ * ------------------------------------------------------------------------- */
+
+/* Icons */
+@import "icons/_mixins.pcss";
+
+/* Typography */
+@import "typography/_mixins.pcss";
```

Comment:
Curious, do we even need to import mixins like this? I thought that [adding the mixinFiles setting](https://github.com/moderntribe/moose/blob/main/postcss.config.js#L58) in the postcss config would auto-locate any mixin partials in any entrypoint.

Reply:
🤦  I missed that, indeed you're right we don't need mixins imported. I've removed it.